doc/CHANGES.md

```diff
@@ -2,12 +2,20 @@
 
 ## Highlighted Features
 
+* Automatically apply `missing_email` tag to bodies who are missing a request
+  email so that they can be found in a public list (Gareth Rees)
 * Protect mass-tag update buttons in admin bodies lists (Gareth Rees)
 
 ## Highlighted Pro Features
 
 ## Upgrade Notes
 
+* _Optional:_ Bodies missing a request email will automatically get tagged
+  `missing_email` as they are updated. If you want to automatically tag them all
+  in one go, run the following from the app root directory:
+
+    bin/rails runner "PublicBody.without_request_email.each(&:save)"
```

Comment:
Render as a code block with:

```suggestion
      bin/rails runner "PublicBody.without_request_email.each(&:save)"
```

Reply:
I think it's correct as-is – 4 spaces from the start of the line, not relative to the lines above, which are already indented 2 because of the bullet point?

<img width="501" alt="Screenshot 2022-09-01 at 16 10 43" src="https://user-images.githubusercontent.com/282788/187949267-9f3bc7ff-0e07-4eee-ad9f-6d1679014b00.png">

Reply:
Oh looks like GitHub prefers your suggestion. Will fix on merge.